presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java

```diff
@@ -66,6 +66,11 @@ public void close()
         sqlExecutor.execute("DROP TABLE " + name);
     }
 
+    public static String getTestTableName(String baseTableName)
```

Comment:
Actually, do we need to pass any argument here? Maybe we could have only `getUniqueTableName()` or just `getUniqueName()`?

Reply:
> Maybe we could have only getUniqueTableName() or just getUniqueName()?

that won't work. table name prefix is very important. it's sometimes the only useuful information in an err msg 

but i think we don't need this method; the inline version with manual concatenation is rather readable

Reply:
I can switch them to match the existing tests. My only preference for this was the extra `_` between the prefix and the random suffix.

Reply:
Just removed that method, opting for the inline string concat like the existing tests.